guidelines/index.html

```diff
@@ -1586,23 +1586,23 @@ <h1 id="title" class="title">Web Content Accessibility Guidelines (WCAG) 2.2</h1
+        <li><strong>スペースがある:</strong> 小サイズ (24 × 24 CSS ピクセル未満) のターゲットにおいて、各ターゲットの<a href="#dfn-bounding-boxes" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-bounding-boxes-1" title="[New]">バウンディングボックス</a>の中心に直径 24 CSS ピクセルの円を置いた場合、その円が別のターゲットもしくは別の小サイズのターゲットの円と交差しないように、ターゲットが配置されている。</li>
+        <li><strong>同等のものが存在する:</strong> その機能を、同一ページ上の、この達成基準を満たす別のコントロールによって、実行することができる。</li>
+        <li><strong>インラインである:</strong> そのターゲットが文中に存在する、又は、そのサイズがターゲットでないテキストの行送りによって制約される。</li>
+        <li><strong>ユーザエージェントのコントロールである:</strong> ターゲットのサイズが<a href="#dfn-user-agents" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-user-agents-3" title="ウェブコンテンツを取得して利用者に提示するあらゆるソフトウェア。">ユーザエージェント</a>によって定められており、かつコンテンツ制作者が変更していない。</li>
+        <li><strong>必要不可欠:</strong> そのターゲットを特定の方法で<a href="#dfn-presentation" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-presentation-4" title="利用者が知覚できる形式でコンテンツをレンダリングすること。">提示</a>することが、情報伝達にとって<a href="#dfn-essential" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-essential-15" title="もし取り除いてしまうと、コンテンツの情報又は機能を根本的に変えてしまい、かつ、適合する他の方法では情報及び機能を実現できない。">必要不可欠</a>である、又は法的に要求されている。</li>
     </ul>
-    <div class="note" role="note" id="issue-container-generatedID-37"><div role="heading" class="note-title marker" id="h-note-37" aria-level="5"><span>注記 1</span></div><p class="">Targets that allow for values to be selected spatially based on position within the target are considered one target for the purpose of the success criterion. Examples include sliders, color pickers displaying a gradient of colors, or editable areas where you position the cursor.</p></div>
-    <div class="note" role="note" id="issue-container-generatedID-38"><div role="heading" class="note-title marker" id="h-note-38" aria-level="5"><span>注記 2</span></div><p class="">For inline targets the line-height should be interpreted as perpendicular to the flow of text. For example, in a language displayed vertically, the line-height would be horizontal.</p></div>
+    <div class="note" role="note" id="issue-container-generatedID-37"><div role="heading" class="note-title marker" id="h-note-37" aria-level="5"><span>注記 1</span></div><p class="">ターゲット内の位置に基づいて空間的に値を選択できるターゲットは、達成基準の目的上、一つのターゲットとみなされる。例としては、スライダー、色のグラデーションが表示されるカラーピッカー、あるいはカーソルを置いたときにそこが編集可能になる領域、などが含まれる。</p></div>
+    <div class="note" role="note" id="issue-container-generatedID-38"><div role="heading" class="note-title marker" id="h-note-38" aria-level="5"><span>注記 2</span></div><p class="">インラインのターゲットの場合、行送りはテキストの流れに対して垂直なものと解釈されるべきである。例えば、縦書きで表示される言語においては、行送りは水平方向となる。</p></div>
```

Comment:
horizontalを水平方向とするならば、verticallyを垂直方向とするのはどうでしょう

```suggestion
    <div class="note" role="note" id="issue-container-generatedID-38"><div role="heading" class="note-title marker" id="h-note-38" aria-level="5"><span>注記 2</span></div><p class="">インラインのターゲットの場合、行の高さはテキストのフローに対して垂直方向として解釈されるべきである。例えば、縦書きで表示される言語では、行の高さは水平方向となる。</p></div>
```